static/app/views/explore/tables/aggregatesTable.tsx

```diff
@@ -93,11 +90,20 @@ export function AggregatesTable({}: AggregatesTableProps) {
       <Table style={tableStyles}>
         <TableHead>
           <TableRow>
-            {fields.map((field, i) => (
-              <TableHeadCell key={i} isFirst={i === 0}>
-                {field}
-              </TableHeadCell>
-            ))}
+            {fields.map((field, i) => {
+              // Hide column names before alignment is determined
+              if (result.isPending) {
+                return <TableHeadCell key={i} isFirst={i === 0} />;
+              }
+
+              const fieldType = meta.fields?.[field];
+              const align = fieldAlignment(field, fieldType);
+              return (
+                <TableHeadCell align={align} key={i} isFirst={i === 0}>
+                  <span>{field}</span>
```

Comment:
Is the extra `span` necessary here?

Reply:
Yeah, the overflow styling in `GridHeadCell` is targeted towards spans/divs and links